### ISSUE TITLE
Swagger link on API webpage broken

### ISSUE THREAD

**Issue by improwise** [abpframework/abp#20835]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When browsing the "webpage" of the API, the link to the swagger definition is incorrect, as it is lacking the /swagger part at the end. It just links to the page currently displayed. 

![image](https://github.com/user-attachments/assets/37b6bcee-69c9-4caa-8f30-cae8d7afbd21)

### Version

8.3

### User Interface

Angular

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

**Comment by 102898Is** [comment]
download
https://bit.ly/4gwC82r
I put the necessary dlls in the archive

**Comment by maliming** [comment]
hi

Project Type?
Reproduction Steps?

**Comment by improwise** [comment]
> hi
> 
> Project Type? Reproduction Steps?

From what we have seen, basically every project type that creates an API which you an browse, but we have mainly been using the Angular and Blazor ones, and all these have this behavior.  Guessing there is probably just some "/swagger" missing somewhere in the code as I assume you are supposed to be able to click and see the Swagger definition.

**Comment by maliming** [comment]
https://github.com/abpframework/abp/issues/20864#issuecomment-2380351975

**Comment by maliming** [comment]
The `HomeController` will redirect to `/swagger`.

```cs
public class HomeController : AbpController
{
    public ActionResult Index()
    {
        return Redirect("~/swagger");
    }
}
```

**Comment by improwise** [comment]
@maliming 

At least when it comes to solutions created with ABP Studio this problem seem to still remain, ie when you click on the Swagger icon/link, you still remain on the same page rather than going to the Swagger page (which I assume is the idea).

![Image](https://github.com/user-attachments/assets/7aa09326-f537-48d2-8093-87a25427ab25)

**Comment by maliming** [comment]
What is your Studio version and your project info?

**Comment by improwise** [comment]
> What is your Studio version and your project info?

Using latest ABP Studio 0.9.22 I believe and noticed this in a newly created layered solution with Angular. Please note that we are using ABP Commercial template, but I would imagine that does not matter in this case (or at least it should not matter).

**Comment by maliming** [comment]
Sorry for that. I have fixed it, and it will be no problem in `0.9.23`.

![Image](https://github.com/user-attachments/assets/8d59fd8a-5bdb-4561-8a1d-a9b05cc343c0)

![Image](https://github.com/user-attachments/assets/d4f5b99b-6a37-4d1d-8f34-5ab9c6744124)

**Comment by improwise** [comment]
Thanks.
